be/src/exec/workgroup/scan_task_queue.cpp

```diff
@@ -36,6 +36,98 @@ bool PriorityScanTaskQueue::try_offer(ScanTask task) {
     return _queue.try_put(std::move(task));
 }
 
+/// MultiLevelFeedScanTaskQueue.
+MultiLevelFeedScanTaskQueue::MultiLevelFeedScanTaskQueue() {
+    double factor = 1;
+    for (int i = NUM_QUEUES - 1; i >= 0; --i) {
+        _queues[i].factor_for_normal = factor;
+        factor *= RATIO_OF_ADJACENT_QUEUE;
+    }
+
+    int64_t time_slice = 0;
+    for (int i = 0; i < NUM_QUEUES; ++i) {
+        time_slice += LEVEL_TIME_SLICE_BASE_NS * (i + 1);
+        _queues[i].level_time_slice = time_slice;
+    }
+}
+
+void MultiLevelFeedScanTaskQueue::close() {
```

Comment:
will this method be frequently invoded? If so, check `_is_closed` outside lock first to improve performance

Reply:
No, it is invoked only when the BE process is shutdowning.